src/huggingface_hub/serialization/_torch.py

```diff
@@ -32,6 +32,26 @@
     import torch
 
 
+def parse_version(version_string):
+    # Extract just the X.Y.Z part from the version string
+    match = re.match(r'(\d+\.\d+\.\d+)', version_string)
+    if match:
+        version = match.group(1)
+        return [int(x) for x in version.split('.')]
+    else:
+        raise ValueError(f"Invalid version string format: {version_string}")
+
+def compare_versions(v1, v2):
+    v1_parts = parse_version(v1)
+    v2_parts = parse_version(v2)
+    return (v1_parts > v2_parts) - (v1_parts < v2_parts)
+
+def torch_version_at_least(min_version):
```

Comment:
Isn't this directly part of `torch`? I remember you showed it to me once. If cannot be imported from `torch`, then I'd prefer to do a 

```py
try:
    from torch.utils._python_dispatch import is_traceable_wrapper_subclass

    # ... method 2.1+
except ImportError:
    # ... method below 2.1
```

I know I said it differently but I don't want to add and maintain this "parse version logic". A simple `ImportError` should be enough for now.

Reply:
I see, yeah this is not part of torch, it's part of our torchao repo.. OK I can change to try/except

Reply:
what about putting these in tests? or do we want to use try/except in tests as well

Reply:
I would try/except in tests as well. For example, make it a private helper `_import_is_traceable_wrapper_subclass` in the package that returns None if not existent and use it in tests. Or another solution you find more elegant as long as it's not adding too many lines of code.

Reply:
sure thanks, updated